src/worldmap/worldmap_sector.cpp

```diff
@@ -353,6 +354,26 @@ WorldMapSector::update(float dt_sec)
       // tux->set_direction(input_direction);
     }
   }
+
+  flush_game_objects();
+}
+
+
```

Comment:
remove extra line

Reply:
I originally added those double lines to separate functions in the class, just like how they are separated in the `.hpp` file. I am not sure whether to remove it, because on one side, this file does this in many places, but otherwise, the rest of the code does not seem to do this.

Reply:
nvm then